test/e2e/tests/notifications/mocks.ts

```diff
@@ -20,18 +20,18 @@ type MockResponse = {
 };
 
 /**
- * E2E mock setup for notification APIs (Auth, Storage, Notifications, Push Notifications)
+ * E2E mock setup for notification APIs (Auth, Storage, Notifications, Push Notifications, Profile syncing)
  *
  * @param server - server obj used to mock our endpoints
  */
-export function mockNotificationServices(server: Mockttp) {
+export async function mockNotificationServices(server: Mockttp) {
   // Auth
   mockAPICall(server, AuthMocks.getMockAuthNonceResponse());
   mockAPICall(server, AuthMocks.getMockAuthLoginResponse());
   mockAPICall(server, AuthMocks.getMockAuthAccessTokenResponse());
 
   // Storage
-  mockAPICall(server, StorageMocks.getMockUserStorageGetResponse());
+  mockAPICall(server, await StorageMocks.getMockUserStorageGetResponse());
```

Comment:
Nice.

@matteoscurati has a PR (unsure if merged) that resolves all this async mocks.

Context - some mocks are now async due to encryption now being async (and how our mocks use realistic encrypted values).

Reply:
These changes will be available in this PR:
https://github.com/MetaMask/metamask-extension/pull/26987